packages/salesforcedx-vscode-lwc/src/commands/forceLightningLwcPreview.ts

```diff
@@ -183,6 +183,7 @@ async function startServer(
     telemetryService.sendCommandEvent(logName, startTime);
   } else if (isDesktop) {
     try {
+      const fullUrl = `${DevServerService.instance.getBaseUrl()}/${DEV_SERVER_PREVIEW_ROUTE}/${componentName}`;
```

Comment:
I think we should add this as a function to the DevServerService - something like getComponentPreviewUrl? I think this will be better in the event that this path ever needs to change and is referenced in more than one location.